packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1649,15 +1649,21 @@ export class ContainerRuntime
 		});
 
 		const loadedFromSequenceNumber = this.deltaManager.initialSequenceNumber;
+		// If the base snapshot was generated when isolated channels were disabled, set the summary reference
+		// sequence to undefined so that this snapshot will not be used for incremental summaries. This is for
+		// back-compat and will rarely happen so its okay to re-summarize everything in the first summary.
+		const summaryReferenceSequenceNumber =
+			baseSnapshot === undefined || metadata?.disableIsolatedChannels === true
```

Comment:
Where does this extra `metadata?.disableIsolatedChannels === true` come from?

Reply:
From ISummaryRuntimeOptions.